Translation/en/Text/Override.txt

```diff
@@ -888,6 +891,10 @@ sr:"速度([\=\+\d\.\%]*)"=Mov Speed $1
 基  础=<size\=16>Basic Info</size>
 基 础 属 性 =<size\=24>Basic Attributes </size>
 人 物 相 关 = Correlation 
+练气解锁=<size\=14>Qi Refining Unlock</size>
+筑基解锁=<size\=14>Foundation Unlock</size>
+化神解锁=<size\=14>God Trans Unlock</size>

```

Comment:
didnt we change that to something else i thing soul formation

Reply:
I can adjust it 1 sec